cluster-up/cluster/kind/common.sh

```diff
@@ -210,9 +195,120 @@ function setup_kind() {
     prepare_config
 }
 
+function ensure_memory_dir_for_etcd_data() {
+    if [ "$BASE_ETCD_DATA_DIR" == none ];then
+        echo "BASE_ETCD_DATA_DIR is not defined"
+        exit 1
+    fi
+    if [ "$ETCD_DATA_DIR" == none ];then
+        echo "ETCD_DATA_DIR is not defined"
+        exit 1
+    fi
+    
+    trap cleanup_dir_for_etcd_data ERR SIGINT SIGTERM
```

Comment:
We can only trigger this is we create it ourselves.

Reply:
I push new commit to skip this if we running on CI